controllers/resources/bentorequest_controller.go

```diff
@@ -1746,6 +1747,16 @@ func (r *BentoRequestReconciler) generateModelSeederPodTemplateSpec(ctx context.
 		}
 	}
 
+	hf_token := ""
+	if envs := opt.BentoRequest.Spec.DownloaderContainerEnv; envs != nil {
+		for _, env := range envs {
+			if env.Name == "HF_TOKEN" {
```

Comment:
‌‌‌‌This way of transmitting the Token in plain text is very insecure. Let's continue using `DownloaderContainerEnvFrom` instead.